packages/apollos-church-api/src/data/content-items/data-source.js

```diff
@@ -203,4 +203,16 @@ export default class ContentItem extends oldContentItem.dataSource {
     });
     return features;
   }
+
+  getBySlug = async (slug) => {
+    const contentItemFromSlug = await this.request('ContentChannelItemSlugs')
+      .filter(`Slug eq '${slug}'`)
+      .get();
```

Comment:
```suggestion
      .first();
```

Reply:
This is nifty. I forgot that this method existed.